crates/uv/src/commands/project/run.rs

```diff
@@ -413,14 +413,19 @@ pub(crate) async fn run(
                 Err(err) => return Err(err.into()),
             };
 
+            let no_install_root = false;
+            let no_install_workspace = false;
+            let no_install_package = vec![];
```

Comment:
It could be nice to capture these all together in a single struct, like BuildOptions. But not blocking.

Reply:
Yes totally, on my mind too when I turned off a clippy warning.

Reply:
https://github.com/astral-sh/uv/issues/6545